docker/vars.local.tests.sh

```diff
@@ -3,6 +3,9 @@
 export PS_DB_URL="jdbc:postgresql://ps_db:5432"
 export PS_DB_OWNER_NAME="postgres"
 export PS_DB_OWNER_PASSWORD="pass_test"
+export PS_DB_HOST="ps_db"
```

Comment:
If we need to have separate variables for db's host and port, then we could remove PS_DB_URL variable and create the url from host and port where needed (in db-connector/build.gradle and application.yml files)